test_settings.py

```diff
@@ -1,2 +1,2 @@
-SQLALCHEMY_DATABASE_URI = "postgresql://postgres:postgres@localhost/postgres"
+SQLALCHEMY_DATABASE_URI = "postgresql://geonatadmin:geonatadmin@localhost:5432/geonature2db"
```

Comment:
```suggestion
SQLALCHEMY_DATABASE_URI = "postgresql://postgres:postgres@localhost/postgres"
```

Ce n’est pas les mêmes paramètres que dans GeoNature dans la CI de ce module.

Reply:
Ah j'avais changé la valeur pour faire les tests en local :man_facepalming: